packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/SettingsDropdown/index.tsx

```diff
@@ -0,0 +1,113 @@
+                horizontalPadding={10}
+                topPadding={0}
+                minWidth={230}
+                masterLink={{
+                    callback: () => {
+                        goto('settings-index');
+                    },
+                    label: <Translation id="TR_ALL" />,
+                    icon: 'ARROW_RIGHT_LONG',
+                }}
+                items={[
+                    {
+                        key: 'settings',
+                        label: <Translation id="TR_SETTINGS" />,
+                        options: [
+                            {
+                                key: '1',
+                                label: <Translation id="TR_APPLICATION" />,
+                                icon: 'APP',
+                                callback: () => {
+                                    goto('settings-index');
+                                },
+                                isRounded: true,
+                                'data-test': '@suite/menu/settings-index',
+                            },
+                            {
+                                key: '2',
+                                label: <Translation id="TR_DEVICE" />,
+                                icon: 'DEVICE',
+                                callback: () => {
+                                    goto('settings-device');
+                                },
+                                isRounded: true,
+                                'data-test': '@suite/menu/settings-device',
+                            },
+                            {
+                                key: '3',
+                                label: <Translation id="TR_COINS" />,
+                                icon: 'COINS',
+                                callback: () => {
+                                    goto('settings-coins');
+                                },
+                                isRounded: true,
+                                'data-test': '@suite/menu/settings-coins',
+                            },
+                        ],
+                    },
+                    // TODO - prepared for Learn & Discover section
+                    // {
+                    //     key: 'guide',
+                    //     options: [
+                    //         {
+                    //             key: '4',
+                    //             label: (
+                    //                 <LabelWithIcon>
+                    //                     <Translation id="TR_LEARN_AND_DISCOVER" />
+                    //                 </LabelWithIcon>
+                    //             ),
+                    //             icon: 'LIGHTBULB',
+                    //             iconRight: 'ARROW_RIGHT',
+                    //             isRounded: true,
+                    //             separatorBefore: true,
+                    //         },
+                    //     ],
+                    // },
```

Comment:
I'd be happiest if this could be committed once really needed in the right context. Or at least, please, link an issue in the TODO.